src/config.ts

```diff
@@ -152,11 +152,15 @@ export interface GraphSimulationSettings {
   onStart?: () => void;
   /**
    * Callback function that will be called on every simulation tick.
-   * The value of the argument `alpha` will decrease over time as the simulation "cools down":
-   * `(alpha: number) => void`.
+   * The value of the first argument `alpha` will decrease over time as the simulation "cools down".
+   * If the node moves under a mouse, its data will be passed as the second argument,
```

Comment:
`If the node moves under a mouse, its data ...` → `If there's a node under the mouse pointer, its datum ...`